doc/source/datasets/list.rst

```diff
@@ -0,0 +1,48 @@
+=============
```

Comment:
I'm also reticent to add to the Git history the documentation for a big mistake (the current on-disk format), especially since we're about to obsolete it anyways...